fe/fe-core/src/main/java/org/apache/doris/qe/SimpleScheduler.java

```diff
@@ -195,7 +195,7 @@ public static void addToBlacklist(Long backendID, String reason) {
     }
 
     public static boolean isAvailable(Backend backend) {
-        return (backend != null && backend.isAlive() && !blacklistBackends.containsKey(backend.getId()));
+        return (backend != null && backend.isAvailable() && !blacklistBackends.containsKey(backend.getId()));
```

Comment:
This method maybe used for load process. And for load, the decommission backend is still able to load data.
So in order to minimize the impact, we can only make this judgment for the **query** request.

Reply:
ok